docs/guides/migrate-from-buidler.md

```diff
@@ -0,0 +1,180 @@
+# Migrating from Buidler
+
+Hardhat is the new and evolved version of Buidler.
+
+This guide will teach you how to migrate your project from Buidler into Hardhat.
+
+## Installing the Hardhat packages
+
+The first thing you need to do to migrate your project, is installing th new npm packages.
+
+The package `@nomiclabs/buidler` is now `hardhat`. The plugins, which used to have packages like `@nomiclabs/buidler-<name>`,
```

Comment:
```suggestion
The package `@nomiclabs/buidler`will move to `hardhat`. The plugins, which used to have package names like `@nomiclabs/buidler-<name>`,
```

Reply:
Why the "will move"? @fzeoli By the time this gets published it will have happened.